frontend/stubs/functions.py

```diff
@@ -1,12 +1,20 @@
 """Stub file for built in functions"""
+from typing import TypeVar, List, Tuple, Dict
 
+Any = TypeVar("Any")
```

Comment:
Do they all correspond to something that exists in the typing module? As in, if there is something with the same meaning in the typing module, do they have the same name? And if there is something with the same name, does it represent the same thing? I don't think this is the case with Any for example (and that's why you had to add Any2, because otherwise it'd have to be the same type, right).